server/handlers/default.go

```diff
@@ -108,7 +111,21 @@ func atomicUpdateHandler(ctx context.Context, w http.ResponseWriter, r *http.Req
 		logger.Errorf("500 POST error applying update request: %v", err)
 		return errors.ErrUpdating.WithDetail(nil)
 	}
-	return nil
+
+	broadcasterKey := ctx.Value("broadcaster")
+	broadcaster, ok := broadcasterKey.(*dNotifications.Broadcaster)
+	if !ok {
+		return errors.ErrUnknown.WithDetail("could not get notification broadcaster")
+	}
+	sourceRecordKey := ctx.Value("sourceRecord")
+	sourceRecord, ok := sourceRecordKey.(dNotifications.SourceRecord)
+	if !ok {
+		return errors.ErrUnknown.WithDetail("could not get notification sourceRecord")
+	}
+
+	// try to get a username, which may be blank if the req is not authd
+	username, _ := ctx.Value(auth.UserNameKey).(string)
+	return notifications.Notify("GUN update", gun, sourceRecord, username, r, broadcaster)
```

Comment:
since it's possible for this to error, should the notification be best effort (and we log appropriately if the notification failed)?